doc/design.md

```diff
@@ -130,6 +130,28 @@ Covers:
 
 Needs: impl, utest
 
+## Restrict Libraries to the Ones Available to Exasol UDFs
```

Comment:
This is a constraint, not a requirement. So you should add a constraints section. See OFT templates for an example.